src/server.js

```diff
@@ -133,7 +133,8 @@ class Server extends EventEmitter {
       this.instrumenter.timeHit(startDate);
       const result = status.isAllowed ? 'accepted' : 'rejected';
       const matchType = Server.getMatchType(rule);
-      this.instrumenter.countHit(result, matchType);
+      const ruleLabel = rule && rule.label ? rule.label : '';
```

Comment:
nit: what's the order of operations here? can you clarify with parens?
```
const ruleLabel = (rule && rule.label) ? rule.label : '';
const ruleLabel = rule && (rule.label ? rule.label : '');
```
assuming it's the first since, but not clear from reading

Reply:
Can do. Ternary is pretty much lowest precedence operator in JS so its not uncommon to see 'naked' expressions like this ahead of it. (In fact your suggestion would be rejected by [rules/no-extra-parens](https://eslint.org/docs/rules/no-extra-parens) but we don't enable it.)

Reply:
d'oh! no need to include for my sake if it's an antipattern.

Reply:
i think its fine, happy to follow the "if someone asked, its probably worth doing/clarifying" school of thought here :)